package/libsettings/libsettings.mk

```
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-LIBSETTINGS_VERSION = v0.1.11
+LIBSETTINGS_VERSION = dc8db22dc803e69b4508005165fb218e2607a39a
```

once release is finalized of course
```suggestion
LIBSETTINGS_VERSION = v0.1.12
```